solr/contrib/ltr/src/java/org/apache/solr/ltr/rest/ManagedModelStore.java

```diff
@@ -190,20 +155,22 @@ public LTRScoringAlgorithm makeLTRScoringAlgorithm(Map<String,Object> map)
     final List<Feature> features = new ArrayList<>();
     final List<Normalizer> norms = new ArrayList<>();
     for (final Object modelFeature : featureList) {
-      try {
-        // check the declared features exist in the feature store
-        final Map<String,Object> modelFeatureMap = 
-            (Map<String,Object>) modelFeature;
-        final Feature feature = parseFeature(modelFeatureMap,
-            fstore);
-        final Normalizer norm = parseNormalizer(modelFeatureMap);
-        norms.add(norm);
-        features.add(feature);
-      } catch (FeatureException e) {
-        throw new SolrException(ErrorCode.BAD_REQUEST, e);
-      } catch (final CloneNotSupportedException e) {
-        throw new SolrException(ErrorCode.BAD_REQUEST, e);
+      final Map<String,Object> modelFeatureMap =
+          (Map<String,Object>) modelFeature;
+
+      final String featureName = (String) modelFeatureMap.get(CommonLTRParams.FEATURE_NAME);
+      final Feature feature = (featureName == null ? null : fstore.get(featureName));
```

Comment:
Could you clarify which null check you mean (the github interface is confusing to me at least)? The featureName==null check before the comment (it is good to have I think since deliberately looking up null in the store seems odd) or the feature==null check after the comment (the existing parseFeature code throws in that case)?